packages/apps/plugins/plugin-navtree/src/components/NavBarStart.tsx

```diff
@@ -35,7 +28,7 @@ export const NavBarStart = ({ activeNode, popoverAnchorId }: { activeNode: Node;
         <NavTreeItemAction
           variant='plank-heading'
           label={t('node actions menu invoker label')}
-          actions={activeNode.actions}
+          actions={activeNode.actions.flatMap((action) => ('invoke' in action ? [action] : []))}
```

Comment:
Very interesting pattern, eager to try it out myself.

Reply:
We should probably introduce the equivalent of graph's `isAction` for the tree actions. I'd like to start using Effect Schema for those as well to make the validation automatic and not a patchwork of object sniffing.